CHANGES_NEXT_RELEASE

```diff
@@ -1 +1 @@
-
+- Add arcgis_connectionTimeout and arcgis_readTimeout sink options to allow set non infinite conection timeouts with arcgis (#2440)
```

Comment:
```suggestion
- [cygnus-ngsi][cygnus-comon] Add arcgis_connectionTimeout and arcgis_readTimeout sink options to allow set non infinite conection timeouts with arcgis (#2440)
```

Based in that the PR touches files in cygnus-ngsi/ and cygnus-common/ directories